examples/gnrc_tcp_cli/Makefile

```diff
@@ -53,7 +53,7 @@ CFLAGS += -DCYCLES=25000
 # development process:
 #CFLAGS += -DDEVELHELP
 
-CFLAGS_OPT = -O0
+#CFLAGS_OPT = -O0
```

Comment:
These are Artifacts to Speedup Testing. I'll clean the makefiles Thursday morning ;)